### ISSUE TITLE
[Inputstream] JSON-RPC request throws if inputstream.adaptive is missing

### ISSUE THREAD

**Issue by awglaat** [asciidisco/plugin.video.netflix#166]
Hi after having a clean UBUNTU 16.04 i added latest KODI ppa and installed KODI.

`
sudo add-apt-repository ppa:team-xbmc/xbmc-nightly
sudo apt-get update
sudo apt-get install kodi
`
After this i installed widevine 
`
wget http://nmacleod.com/public/libreelec/getwidevine.sh
ARCH=x86_64 sh getwidevine.sh
`
Thinking this was sufficient i have downloaded the repo and did an install of the netflix plugin. 
After adding username and password it didnt start (check error log).

after installing pycryptodomex using 
`
pip install pycryptodomex 
`
and checking dependencies are met with 
`
pip2 install pycryptodomex 
`

I was able to start Netflix and login, browse but when selecting start NO VIDEO. 

even a uninstall remove of .kodi and fresh install didnt help.

error i`m not facing in log file is:

```
.176 T:140340233549568   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.TypeError'>
                                            Error Contents: 'NoneType' object is not iterable
                                            Traceback (most recent call last):
                                              File "/home/hp-server/.kodi/addons/plugin.video.netflix/addon.py", line 37, in <module>
                                                navigation.router(paramstring=sys.argv[2][1:])
                                              File "/home/hp-server/.kodi/addons/plugin.video.netflix/resources/lib/utils.py", line 27, in wrapped
                                                result = f(*args, **kwargs)
                                              File "/home/hp-server/.kodi/addons/plugin.video.netflix/resources/lib/Navigation.py", line 54, in router
                                                return self.open_settings(params['url'])
                                              File "/home/hp-server/.kodi/addons/plugin.video.netflix/resources/lib/Navigation.py", line 622, in open_settings
                                                (is_addon, _) = self.kodi_helper.get_inputstream_addon()
                                            TypeError: 'NoneType' object is not iterable
                                            -->End of Python script error report<--
15:39:45.608 T:140342674712960   ERROR: Control 55 in window 10025 has been asked to focus, but it can't
15:39:48.412 T:140340233549568   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.TypeError'>
                                            Error Contents: 'NoneType' object is not iterable
                                            Traceback (most recent call last):
                                              File "/home/hp-server/.kodi/addons/plugin.video.netflix/addon.py", line 37, in <module>
                                                navigation.router(paramstring=sys.argv[2][1:])
                                              File "/home/hp-server/.kodi/addons/plugin.video.netflix/resources/lib/utils.py", line 27, in wrapped
                                                result = f(*args, **kwargs)
                                              File "/home/hp-server/.kodi/addons/plugin.video.netflix/resources/lib/Navigation.py", line 128, in router
                                                self.play_video(video_id=params['video_id'], start_offset=params.get('start_offset', -1), infoLabels=params.get('infoLabels', {}))
                                              File "/home/hp-server/.kodi/addons/plugin.video.netflix/resources/lib/utils.py", line 27, in wrapped
                                                result = f(*args, **kwargs)
                                              File "/home/hp-server/.kodi/addons/plugin.video.netflix/resources/lib/Navigation.py", line 165, in play_video
                                                return self.kodi_helper.play_item(esn=esn, video_id=video_id, start_offset=start_offset, infoLabels=infoLabels)
                                              File "/home/hp-server/.kodi/addons/plugin.video.netflix/resources/lib/KodiHelper.py", line 829, in play_item
                                                (inputstream_addon, inputstream_enabled) = self.get_inputstream_addon()
                                            TypeError: 'NoneType' object is not iterable
                                            -->End of Python script error report<--
15:39:48.440 T:140342674712960   ERROR: Playlist Player: skipping unplayable item: 0, path [plugin://plugin.video.netflix/?action=play_video&infoLabels=%7B%27rating%27%3A+0%2C+%27writer%27%3A+%27%27%2C+%27episode%27%3A+1%2C+%27plot%27%3A+u%27Wanneer+een+lading+cannabis+van+de+Marokkaanse+drugsbaas+El+Feo+wordt+gestolen%2C+heeft+de+diefstal+repercussies+in+drie+landen.%27%2C+%27season%27%3A+1%2C+%27tvshowtitle%27%3A+u%27Cannabis%27%2C+%27title%27%3A+u%27Aflevering+1%27%2C+%27mediatype%27%3A+u%27episode%27%2C+%27mpaa%27%3A+u%27%27%2C+%27director%27%3A+%27%27%2C+%27genre%27%3A+u%27Dramaseries%27%2C+%27year%27%3A+2016%2C+%27duration%27%3A+2902%2C+%27playcount%27%3A+%27%27%7D&video_id=80115033&start_offset=2871]
```

anybody can provide some more details on this?

**Comment by asciidisco** [comment]
Looks like something is wrong with your `inputstream.adaptive` adaptive plugin.

Is it activated? 
Do you have the correct binary for your system?

Have you executed all the steps described here?!:

- use following [gist](https://gist.github.com/ruario/3c873d43eb20553d5014bd4d29fe37f1) to install latest `widevine`

- Install Kodi inputstream-adaptive and copy libwidevinecdm.so to Kodi installation directory

```bash
sudo apt-get install kodi-inputstream-adaptive
cp /opt/google/chrome/libwidevinecdm.so /home/$USER/.kodi/cdm/
```
- Enable adaptive plugin in `System > Add-Ons > My Add-Ons > VideoPlayer InputStream > InputStream Adaptive`

**Comment by asciidisco** [comment]
Fixed in `master`will be in relese 0.12.1
